src/ui-misc.cc

```diff
@@ -1626,4 +1626,15 @@ gboolean defined_mouse_buttons(GtkWidget *, GdkEventButton *event, gpointer data
 	return ret;
 }
 
+GdkPixbuf *gq_gtk_icon_theme_load_icon_copy(GtkIconTheme *icon_theme, const gchar *icon_name, gint size, GtkIconLookupFlags flags)
+{
+	GError *error = nullptr;
+	GdkPixbuf *icon = gtk_icon_theme_load_icon(icon_theme, icon_name, size, flags, &error);
+	if (error) return nullptr;
```

Comment:
@qarkai 
Should there be a g_error_free() here at line 1633?

Reply:
You are right. Created #1429.